### PR TITLE
feat(contract): Truffle dashboard

### PR DESCRIPTION
**Pull request by Abhishekkochar**

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -82,7 +83,7 @@ const config: HardhatUserConfig = {
       allowUnlimitedContractSize: true,
     },
     DMCTestnet: {
-      url: 'http://35.187.53.161:20551',
+      url: 'https://testnet-dmc.mydefichain.com:20551',
       accounts: process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
```
[the remark this answers is not in the document]
Comment (cuongquangnam) [reply hardhat.config.ts:87]: Just write out my finding here, the url in this configuration does not matter when we use truffle for deployment.
What matters is the configuration in metamask ( and the chain id in the metamask wallet should match the chain id in the etherscan field configuration)

File: scripts/dataFeed.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+async function dataFeed(contractAddress: string) {
+  const [signers] = await ethers.getSigners();
+  const provider = new ethers.providers.JsonRpcProvider('http://13.214.74.236:20551/');
+  const stateContract = new ethers.Contract(contractAddress, StateRelayer__factory.abi, provider);
+  const tx = await stateContract
+    .connect(signers)
+    .updateMasterNodeInformation(['1528620054717725800', '10636', '917', '3491', 18], { gasLimit: 100000 });
+  await tx.wait();
+}
+const stateRelayerProxyAddress = '0xAE105DE0afC82f91ddBF97cf2197dbd4627a8D16';
```
Comment (cuongquangnam) [inline scripts/dataFeed.ts:20]: what is this file for?

Comment (Abhishekkochar) [reply scripts/dataFeed.ts:20]: This file solely for testing that if you run the script with -- truffleDashBoard, User will be asked to confirm the tx on metamask.

**Review by cuongquangnam — COMMENTED** [review]
Except for the dataFeed script, everything else looks fine to me

File: scripts/dataFeed.ts (hunk 2 above)
Comment (Abhishekkochar) [reply scripts/dataFeed.ts:20]: Just for comments here. 

When adding using `--network truffleDashboard`, tx will be deployed via metamask.

Comment (cuongquangnam) [reply scripts/dataFeed.ts:20]: yeah, I tested it out, haha

**Comment by Abhishekkochar** [comment]
> Except for the dataFeed script, everything else looks fine to me

Done!

**Review by cuongquangnam — APPROVED** [review]

**Pull request merged**